puphpet/config.yaml.template

```diff
@@ -79,25 +100,31 @@ apache:
     settings:
         user: www-data
         group: www-data
-        default_vhost: true
+        default_vhost: false
```

Comment:
Apache's default vhost maps to /var/www, and messes stuff up if it's active (it will handle requests where the vhost name doesn't match, which is bad).

Sent from my iPhone

> On Jan 31, 2015, at 8:50 AM, Andrew Lechowicz <notifications@github.com> wrote:
> 
> In puphpet/config.yaml.template:
> 
> > @@ -79,25 +100,31 @@ apache:
> >      settings:
> >          user: www-data
> >          group: www-data
> > -        default_vhost: true
> > +        default_vhost: false
> @ricog is this correct?
> 
> —
> Reply to this email directly or view it on GitHub.
>